ProjectPlant-HW/documentation/bom.md

```diff
@@ -0,0 +1,89 @@
+# Bill of Materials
+
+## Introduction
+
+This document will outline the total list of parts required for each variation/version of the hardware, including testers and intermediate revisions. This document will only detail the materials required. Connection and operation will be left to the design document.
+
+## Testers
+### Bluetooth Tester
+
+* Arduino Uno/Nano
+* HC-05 Bluetooth Module
+* 5v-3.3v level converter module
+* M-F Dupont wires
+
+### DHT22 Tester
+
+* Arduino Uno/Nano
+* DHT22 Humidity/Temperature Sensor
+* M-F Dupont wires
+
+### LDR Tester
+
+* Arduino Uno/Nano
+* 5Kohm Light Dependent Resistor (CdS cell)
+* 5.6Kohm 5% 1/4W resistor
+* M-F Dupont wires
+* Light duty hookup wire
+
+### Moisture Tester
+
+* Arduino Uno/Nano
+* DFRobot resistive moisture sensor 
+
+## "Spaghetti" version
+
+Note: this is the first cut of the hardware design prior to using a breadboard to begin simplifying the setup.
+
+* Arduino Uno/Nano
+* HC-05 Bluetooth Module
+* 5v-3.3v level converter module
+* DHT22 Humidity/Temperature Sensor
+* 5Kohm Light Dependent Resistor (CdS cell)
+* 5.6Kohm 5% 1/4W resistor
+* DFRobot resistive moisture sensor
+* M-F Dupont wires
+* Light duty hookup wire
+
+## "Breadboard" version
+
+Note: This is the first cut of the hardware design, using a breadboard to simplify the "spaghetti" wiring first used"
+
+* Arduino Uno/Nano
+* HC-05 Bluetooth Module
+* 5v-3.3v level converter module
+* DHT22 Humidity/Temperature Sensor
+* 5Kohm Light Dependent Resistor (CdS cell)
+* 5.6Kohm 5% 1/4W resistor
+* DFRobot resistive moisture sensor
+* M-F Dupont wires
+* Light duty hookup wire
+* Small breadboard (2x power rails, 2 sides of 5x30 tie points)
+
+## "Naked MCU" version

```

Comment:
Naked MCU? I get what you mean but in the future I think its best to stay away from acronyms in titles.